src/tiles/grid/item-tile.ts

```diff
@@ -40,9 +41,12 @@ export class ItemTile extends LitElement {
             .loggedIn=${this.loggedIn}
             .isCompactTile=${false}
             .isListTile=${false}
+            .hasSnippets=${this.hasSnippets}
```

Comment:
```suggestion
            style=${this.hasSnippets? '--imageBlockHeight: 11rem;' : ''}
```
something like this.. see my comment in item-tile about component scope.